### ISSUE TITLE
Representing Giveth External Comms

### ISSUE THREAD

**Issue by c0ric0ri** [Giveth/giveth-planning#872]
As the team grows and different members attend events to give talk about Giveth, it makes sense to align on messaging and also coordinate communications strategies to avoid redundancies. 

The idea is to have some resources created that outline key information and the process for preparing for talks.

Some points already brainstormed:

- representatives should be aware of roadmap, have a version of how to represent Giveth
- we need to review process for major talks at conferences
- we should coordinate to avoid speaking redundantly
- if you are giving talks, reach out to Lauren to coordinate
- we should set up support materials in Notion
- we could have a blank slide template that people can use to create talks
- Let's use this GIthub issue to compile info for talks, and then get it set up in Notion

**Comment by c0ric0ri** [comment]
https://www.notion.so/giveth/Giveth-essentials-1df4b43241604bf18ce57524e3e3a1d9

**Comment by c0ric0ri** [comment]
Hey @danelsuga @oyealmond how are things going here? With DevCon coming up fast it would be great if we can start reviewing drafts of support materials today/Monday

**Comment by danelsuga** [comment]
It's Friday night, so I won't be able to provide anything by Monday. I am trying to set up a sync with @oyealmond. That's all I've got.

**Comment by danelsuga** [comment]
I am at my niece's funeral tonight.

**Comment by c0ric0ri** [comment]
Hey @oyealmond @danelsuga 

So actually this is not specific to Bogota, the idea of this issue is actually beyond these two individual talks. As the team is growing quickly and events are always on the horizon (digital and physical), it's important that we have these resources to ensure aligned representation of Giveth externally. It's a need that has been expressed and that we've identified.

For example the feedback we've received [here](https://docs.google.com/document/d/1mdtBMCvdLqE4Lzd8jY_aFwFIpcGMc4N_45uUOyQH4bA/edit) is:

"Is there anything you see that Giveth can be doing better as an organization? How can Giveth do better?

- Too many different ideas about what problem Giveth is trying to solve
- Giveth is very complex and very difficult to explain to a citizen sector organization"

It would great to see information compiled here in this issue, for example (just brainstorming here):

- helpful talks and articles that represent Giveth well (also the messaging doc we created for Barcelona)
- graphics representing the roadmap
- documents addressing common Q&As

And as we discussed in the group, it makes sense for these resources to be set up in Notion so they can be easily accessed by contributors and updated when needed.

Does that make sense?

**Comment by danelsuga** [comment]
Thanks, @c0ric0ri.
@oyealmond and I are syncing today.

**Comment by oyealmond** [comment]
Our next steps for this, making a "shilling Giveth" guide, and presenting on a virtual session with the team involved

**Comment by danelsuga** [comment]
Will do asap! Super awesome @oyealmond !

**Comment by danelsuga** [comment]
This is looking great.

**Comment by danelsuga** [comment]
It would be great to set up another "Giveth Essentials" presentation as an AMA for core team to learn about the roadmap and how to answer some tough questions when onboarding new folks.

**Comment by oyealmond** [comment]
Love it! @laurenluz would you be down to give a Giveth essentials session of how to represent Giveth next Friday October 21 at 8am Costa Rica Time? The theme is "how to shill Giveth" and this is the draft of the agenda we can follow during the session https://docs.google.com/document/d/1pzVEZGMLA4EgA892TIv5aTxm2aZ0ItmkxTFNwEjgjRQ/edit?usp=sharing 

I think We will have more than enough time to invite the team that's representing Giveth , and talking about Giveth

**Comment by danelsuga** [comment]
@NikolaCreatrix you might want to check this out too! Related to #761

**Comment by oyealmond** [comment]
yes! We're having the first one this Friday 8am Costa Rica time :) @danelsuga 
Ty @laurenluz & @franculio !

**Comment by danelsuga** [comment]
Ok. 

Where is the info?

**Comment by danelsuga** [comment]
Friday 16h00 CET
@laurenluz 

Thanks @oyealmond !

**Comment by franculio** [comment]
Hey @danelsuga ! here's the discord event! Sorry for the delay https://discord.com/events/679428761438912522/1031986648222806137

**Comment by WhyldWanderer** [comment]
I had a chance to review the guide document @oyealmond! It looks good to me...


I look forward to seeing you guys on Friday. If I think of any other questions, Ill make sure to put them in the doc.. I know I had some faq's from devcon also

**Comment by danelsuga** [comment]
Can some one drop the Giveth Essentials session from last week here please?
And maybe those links can be kept together in the future, maybe on a forum post or something?
Then maybe this issue can be closed.
Thanks! @oyealmond @c0ric0ri

**Comment by franculio** [comment]
Here's the link @danelsuga . You can find all of the recordings in the Giveth Transparency channel on youtube but I think the idea is to keep the links in the notion page https://www.notion.so/giveth/Giveth-essentials-1df4b43241604bf18ce57524e3e3a1d9

https://www.youtube.com/watch?v=KIXLu_f4GJg

**Comment by oyealmond** [comment]
hey @danelsuga ! 
You're so right. 
[Here's the document that was made that includes the session and more. ](https://www.notion.so/Representing-Giveth-172f576215544f608a109d3d2d7b48bb)Thank you!
